mLRS/Common/sx-drivers/sx126x_driver.h

```diff
@@ -160,6 +160,19 @@ class Sx126xDriverCommon : public Sx126xDriverBase
         if (osc_configuration != SX12xx_OSCILLATOR_CONFIG_CRYSTAL) {
             SetDio3AsTcxoControl(osc_configuration, 250); // set output to 1.6V-3.3V, ask specification of TCXO to maker board
         }
+
+        //Image Calibration per datasheet 9.2.1
```

Comment:
blank after //
in the next lines first char lower case pl :)

Reply:
Updated.

Reply:
THX